fcl/dunefd/caf/standard_cafmaker_dune10kt_1x2x6.fcl

```diff
@@ -0,0 +1,46 @@
+#include "services_dune.fcl"
+#include "mvamodule.fcl"
+#include "CAFMaker.fcl"
+
+process_name: CafMaker
+
+services:
+{
+  # Load the service that manages root files for histograms where the CAF info will be saved.
+  TFileService: { fileName: "caf.root" }
+  TimeTracker:       {}
+  RandomNumberGenerator: {}
+  MemoryTracker:         {}
+  message:      @local::dune_message_services_prod_debug #TODO: Change for prod
+  FileCatalogMetadata:  @local::art_file_catalog_mc
+  @table::dunefd_simulation_services
```

Comment:
This should use the 1x2x6 service block.
Should this use the reco services or the simulation services?

Reply:
Do you have a specific example? I admit I never really know which of those have to be used in which circumstances.

Reply:
`@table::dunefd_1x2x6_reco_services` I guess